tanner/server.py

```diff
@@ -126,20 +128,28 @@ def create_app(self, loop):
         return app
 
     async def start_background_delete(self, app):
-        app['session_delete'] = asyncio.ensure_future(self.delete_sessions())
+        app["session_delete"] = asyncio.ensure_future(self.delete_sessions())
 
     async def cleanup_background_tasks(self, app):
-        app['session_delete'].cancel()
-        await app['session_delete']
+        app["session_delete"].cancel()
+        await app["session_delete"]
 
     def start(self):
         loop = asyncio.get_event_loop()
-        self.redis_client = loop.run_until_complete(redis_client.RedisClient.get_redis_client())
+        self.redis_client = loop.run_until_complete(
+            redis_client.RedisClient.get_redis_client()
+        )
+        self.pg_client = loop.run_until_complete(
```

Comment:
Maybe we can do it like aiohttp suggests? https://docs.aiohttp.org/en/stable/faq.html#where-do-i-put-my-database-connection-so-handlers-can-access-it